online-inference/tensorizer-isvc/gptj_6b_hf/kserve/requirements.txt

```diff
@@ -0,0 +1,3 @@
+# Not pinned due to tensorizer package resolution 
+kserve
```

Comment:
This should also be changed to the version which pip finds to be suitable with `tensorizer`.